docs/reference/deploy-microsoft-azure.md

```diff
@@ -0,0 +1,80 @@
+---
+id: deploy-microsoft-azure
+title: Deploy on Microsoft Azure Databricks
```

Comment:
This chapter seems a bit outdated

Reply:
Yes, absolutely. Let's merge it as it is and all of the "deactivated" pages will be reviewed or rewritten before they're made visible in the sidebar.